specification/graphrbac/data-plane/Microsoft.GraphRbac/stable/1.6/graphrbac.json

```diff
@@ -2720,7 +2720,8 @@
           "type": "string",
           "format": "date-time",
           "description": "The time at which the directory object was deleted.",
-          "readOnly": true
+          "readOnly": true,
+          "x-nullable": true
```

Comment:
Does a JSON `null` actually come across the wire?

Reply:
> Does a JSON `null` actually come across the wire?

We use it in .NET Track2 test https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/testcommon/Azure.Graph.Rbac
With new version of code gen, when the sdk try to deserialize the service principal with `deletionTimestamp` value null, it will get error: **Value cannot be null**. There is no null check for this property. I've addded you to the mail thread so that you can get the details information.